include/AutomatableModel.h

```diff
@@ -94,6 +94,8 @@ class LMMS_EXPORT AutomatableModel : public Model, public JournallingObject
 	virtual void accept(ModelVisitor& v) = 0;
 	virtual void accept(ConstModelVisitor& v) const = 0;
 
+	static AutomatableModel* s_lastChangedModel;
```

Comment:
I like to use `inline static` when I can to avoid the definition in the .cpp file. It's personal preference mostly.

```suggestion
inline static AutomatableModel* s_lastChangedModel = nullptr;
```